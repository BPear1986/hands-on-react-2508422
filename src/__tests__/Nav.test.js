import React from "react"
import renderer from "react-test-renderer"
import Nav from"../components/Nav" 

const myJSON = require('../cast.json')

it("test runs successfuly", ()=>{
  const truth = true
  expect(truth).toEqual(true)
})

it("renders with all cast members", () => {
  const NavBar =  renderer.create(<Nav cast = {myJSON}/>)
  expect(NavBar.toJSON).toMatchSnapshot()
})