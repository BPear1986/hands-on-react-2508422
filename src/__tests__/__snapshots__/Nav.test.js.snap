// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with all cast members 1`] = `[Function]`;
